src/views/wiki/edition/OutingEditionView.vue

```diff
@@ -10,15 +10,18 @@
           class="is-narrow"
           :document="document"
           :field="fields.date_start"
-          :max="showBothDates ? document.date_end : undefined"
+          :max="currentDate || (showBothDates ? document.date_end : undefined)"
```

Comment:
By doing this, you allow to select 2 dates with the first one being after the last one, because `currentDate` is always set and thus the second part is never evaluated.

Instead, it should be `currentDate` if single date, but the min of `document_date_end` and `currentDate` otherwise.

Reply:
Good catch!
I think a clean condition could be: `showBothDates ? document.date_end : currentDate` as we know that `document.date_end < currentDate`